Storage/StorageTests/Tools/FileStorageTests.swift

```diff
@@ -19,13 +19,17 @@ final class FileStorageTests: XCTestCase {
     }
 
     func testFileIsLoaded() {
-        XCTAssertNoThrow(try subject?.data(for: fileURL!))
+        var data: [PreselectedProvider]?
+        XCTAssertNoThrow(data = try subject?.data(for: fileURL!))
+        XCTAssertNotNil(data)
     }
 
     func testErrorIsTriggeredWhenFileFailsToLoad() {
         let url = URL(string: "http://somewhere.on.the.internet")
```

Comment:
I believe this can be implemented in a different way since under some circumstances the router or the local machine can return a fake page which shows that this website doesn't exist (and in this case, the test fails). Maybe we can use a local URL that points to a non-existing file?

Reply:
ah yea this sounds familiar! it's logged in this issue with flaky tests https://github.com/woocommerce/woocommerce-ios/issues/1687 (which links to the original discussion https://github.com/woocommerce/woocommerce-ios/pull/1672#pullrequestreview-337143774).

do you have any suggestions for a URL that would always be non-existent? we could use some local URL like `localhost` but it's not impossible developers have some local server running 🤔 (maybe a port that's very unlikely `localhost:0000`?)

Reply:
Thanks @jaclync for finding the links! I opened a proposal here: https://github.com/woocommerce/woocommerce-ios/pull/2330 I think in this way tests always work locally.